aptos-move/framework/src/natives/aggregator_natives/aggregator.rs

```diff
@@ -18,6 +21,18 @@ use move_vm_types::{
 use smallvec::{smallvec, SmallVec};
 use std::collections::VecDeque;
 
+/// The value of aggregator overflows. Raised by native code.
+pub(crate) const EAGGREGATOR_OVERFLOW: u64 = 0x02_0001;
+
+/// The value of aggregator underflows (goes below zero). Raised by native code.
+pub(crate) const EAGGREGATOR_UNDERFLOW: u64 = 0x02_0002;
+
+pub(crate) fn abort_error(message: impl ToString, code: u64) -> PartialVMError {
+    PartialVMError::new(StatusCode::ABORTED)
```

Comment:
This was legacy code before safe natives, we can land as is or just return SafeNativeError directly, so no need to call `from`?